selfdrive/car/torque_data/params.yaml

```diff
@@ -43,13 +43,13 @@ KIA K5 2021: [2.405339728085138, 1.460032270828705, 0.11650989850813716]
 KIA NIRO EV 2020: [2.9215954981365337, 2.1500583840260044, 0.09236802474810267]
 KIA SORENTO GT LINE 2018: [2.464854685101844, 1.5335274218367956, 0.12056170567599558]
 KIA STINGER GT2 2018: [2.7499043387418967, 1.849652021986449, 0.12048334239559202]
-LEXUS ES 2019: [2.0203086922726112, 2.134803912579666, 0.12757526789308554]
```

Comment:
not a big deal now, but we'll eventually want something to update the max lat accel. it's used in the docs

Reply:
Yeah, I'll do it in another PR. Ideally it should be the same as the expected max (via max torque).